pandas/tests/io/test_html.py

```diff
@@ -869,6 +881,18 @@ def test_computer_sales_page(self):
         data = os.path.join(DATA_PATH, 'computer_sales_page.html')
         self.read_html(data, header=[0, 1])
 
+    def test_multiple_header_rows(self):
+        # Issue #13434
+        expected_df = DataFrame(data=[("Hillary", 68, "D"),
```

Comment:
Yes, it tests it twice: once using the BeautifulSoup parser and again with lxml.